deploy/example/filestore-pod.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
```

Comment:
Can we also put deployment examples under the gke-nfs-lb directory?

Reply:
added